### ISSUE TITLE
Add FireCapture to the mix

### ISSUE THREAD

**Issue by MattBlack85** [devDucks/astroarch#19]
blocked by https://github.com/devDucks/astroarch-pkgs/issues/4

**Comment by MattBlack85** [comment]
closing, Firecapture is closed source and doesn't provide any aarch64 package
maybe 2.7.11 will add it, but for now it is a won't fix
